libminifi/include/core/state/UpdatePolicy.h

```diff
@@ -69,9 +69,7 @@ class UpdatePolicy {
   }
 
  protected:
-  explicit UpdatePolicy(const UpdatePolicy &other)
-      : enable_all_(other.enable_all_), properties_(other.properties_) {
-  }
+  explicit UpdatePolicy(const UpdatePolicy &other) = default;
```

Comment:
Would you mind fixing the copy and move ctors by removing the `explicit` specifiers and making the move ctor and assignment operators take mutable rvalue reference? Not strictly related to your changes, but a trivial fix nearby.

Reply:
Done.